### PR TITLE
Revert stupid hardcoded format for channels and add config

### PR DESCRIPTION
**Pull request by Yatekii**

### CODE REVIEW

File: src/rttui/app.rs [hunk 1]
```diff
@@ -54,13 +53,15 @@ impl App {
         let mut down_channels = rtt.down_channels().drain().collect::<Vec<_>>();
         if !config.rtt.channels.is_empty() {
             for channel in &config.rtt.channels {
+                println!("{:?}", channel.format);
```
Comment (mvirkkunen) [inline src/rttui/app.rs:56]: Looks like debug logging

Comment (Yatekii) [reply src/rttui/app.rs]: Yep, nice spot, thanks!

**Comment by mvirkkunen** [comment]
I didn't actually try running this, but the changes look legit to me.

**Review by mvirkkunen — APPROVED** [review]

**Pull request merged**